### ISSUE TITLE
Add support for dynamic frameworks

### ISSUE THREAD

**Issue by michalchudziak** [callstack/react-native-brownfield#9]

**Comment by thymikee** [comment]
Sounds like this is unblocked now :)

**Comment by ArekChr** [comment]
It's been open a while without a description, so I wanted to check whether it's still relevant before anyone starts on it.

Reading the current docs and code, things look quite different from 2019: brownfield package:ios now produces XCFrameworks and an optional local SPM package, and the iOS guide lists static pod linkage (use_frameworks! :linkage => :static) as a requirement rather than a limitation.

So:
1. Is dynamic linkage still on the roadmap, or has this been superseded by the current XCFramework/SPM flow?
2. If it's still wanted, is there a specific use case behind it? Knowing that would help scope the work.

**Comment by artus9033** [comment]
Hi all, since the use of dynamic frameworks is a bit niche, we are not going to prioritize this now. If you are facing a need for this, please let us know - we'll reopen and revisit the priority.
